src/secret_sharing/replicated/malicious/additive_share.rs

```diff
@@ -182,6 +183,41 @@ impl<V: ArithmeticShare> Mul<V> for AdditiveShare<V> {
     }
 }
 
+impl<V: ArithmeticShare> Serializable for AdditiveShare<V>
```

Comment:
todo serde macro for these collections so we can hide the crazy size calculations